### ISSUE TITLE
REC1: Understand pros and cons of making source code public from day 1

### ISSUE THREAD

**Issue by JasonJWilliamsNY** [SoftDev4Research/4OSS-lesson#3]
Learners should make an informed decision whether OSS is appropriate for your software. This could be a discussion

**Comment by trallard** [comment]
Following this, I think adding a section on OSS Licensing is **very important**. Especially since a number of people assume that making the code public is enough and people can use it by default, unfortunately, that is not the case.

**Comment by mkuzak** [comment]
@trallard yes, this is the plan, we have the whole episode  "Adopt a licence and comply with the licence of third-party dependencies", it will most likely first episode of the workshop. Please see all the notes from CarpentryCon [here](http://pad.software-carpentry.org/4OSS-carpentrycon) for more details. I'll try to migrate those into gh issues in the coming days.

**Comment by trallard** [comment]
Great thanks! I might have missed the Etherpad as I also missed CarpentryCon 😢 Anyway, there are some really nice suggestions already dropped in there

**Comment by orchid00** [comment]
Going  back to @JasonJWilliamsNY point. I would like to add this pros-cons discussion to the introduction https://github.com/SoftDev4Research/4OSS-lesson/blob/gh-pages/_episodes/01-introduction.md what do you think? @mkuzak @fpsom @allegravia ?

**Comment by orchid00** [comment]
This https://www.st-andrews.ac.uk/media/research-data-management/documents/other/Barriers%20to%20data%20sharing%20data%20with%20suggested%20answers.pdf and this https://docs.google.com/document/d/1EvhqaDlC1u2iVfqGv-lvPGOgTM5Gc0zRbe70OhaBcqM/edit#heading=h.gewa6vf90ds1
are nice to start a conversation about pros and cos
